lib/vagrant-sshfs/cap/linux/sshfs_mount.rb

```diff
@@ -157,8 +164,8 @@ def self.sshfs_slave_mount(machine, opts, hostpath, expanded_guest_path)
           #          stdin <= r2        pipe2         w2 <= stdout 
           #
           # Wire up things appropriately and start up the processes
-          p1 = spawn(sftp_server_cmd,  :out => w2, :in => r1)
-          p2 = spawn(ssh_cmd, :out => w1, :in => r2)
+          p1 = spawn(sftp_server_cmd, :out => w2, :in => r1, :err => t1)
```

Comment:
I think you want to make sure that the new processes get a new process group. This only works OS specific though. See also - https://github.com/vagrant-landrush/landrush/blob/master/lib/landrush/server.rb#L71.

Not sure whether this has a direct effect here, but I several issues with the spawned processes until I properly made sure that they are completely unrelated (not even sharing the process group)